src/events/request/RequestEventHandler.ts

```diff
@@ -91,6 +124,16 @@ export default class RequestEventHandler implements EventHandler<'message'> {
 		}
 	};
 
+	private getTickets( content: string ): Set<string> {
```

Comment:
Why does this return a set, not an array? It will be converted into an array anyway.